### PR TITLE
[Related #88] Toast component and its story

### PR DESCRIPTION
**Pull request by supunTE**

### CODE REVIEW

**Comment by supunTE** [comment]
image
image
image

File: apps/client/src/components/molecules/toast/Toast.component.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            'my-5',
+            {
+              'bg-neutral-40 opacity-40': type === ToastType.Info,
+            },
+            {
+              'bg-primary-70 opacity-40': type === ToastType.Error,
+            },
+            {
+              'bg-secondary-70 opacity-40': type === ToastType.Warning,
+            },
```
Comment (dalanad) [inline apps/client/src/components/molecules/toast/Toast.component.tsx]: Can't we define these in a single object? why  do they have to be separate ones?

**Review by dalanad — CHANGES_REQUESTED** [review]

**Comment by dalanad** [comment]
> image image image

Is the contrast of the error title good? I think it's a bit hard to read

**Review by dalanad — APPROVED** [review]

File: apps/client/src/components/molecules/toast/Toast.component.tsx (hunk 1 above)
Comment (supunTE) [reply apps/client/src/components/molecules/toast/Toast.component.tsx]: Updated

**Pull request merged**